code/game/objects/items/weapons/storage/parachute.dm

```diff
@@ -0,0 +1,44 @@
+//Parachutes
+/obj/item/storage/backpack/parachute
+	name = "parachute"
+	desc = "A specially-made backpack, designed to help one survive jumping from incredible heights. It sacrifices some storage space for that added functionality."
+	icon = 'icons/obj/items/storage/backpack/parachute.dmi'
+	var/packed = TRUE
+
+/obj/item/storage/backpack/parachute/Initialize(ml, material_key)
+	max_storage_space = round(max_storage_space * 0.5)
```

Comment:
I disagree with this as a pattern. It should just have a lower initial storage space.